src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -89,11 +89,17 @@ private function fixIsset(Tokens $tokens, int $index): void
             return; // some weird stuff inside the isset
         }
 
+        $issetCode = $issetTokens->generateCode();
+
+        if ('$this' === strtolower($issetCode)) {
```

Comment:
Personally I am not sure if it should be lowercased. Look at [this example](https://3v4l.org/jOXCK):

```php
class Foo {
    public function bar(): void
    {
        var_dump($this);
        var_dump($THIS);
    }
}

(new Foo())->bar();
```

`$this` is automagically defined and available, while `$THIS` is not. It's a different variable that [can be assigned within class](https://3v4l.org/200Sn) (thanks, PHP 🙄), while [`$this` can't](https://3v4l.org/1jnqZ).

It looks like we should exclude from `ternary_to_null_coalescing` only exact `$this`.